ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -544,16 +554,25 @@ test-suite infra-test
   main-is:        Main.hs
   other-modules:
     Ouroboros.Consensus.Util.Tests
+    Test.Ouroboros.Consensus.ChainGenerator.Tests
+    Test.Ouroboros.Consensus.ChainGenerator.Tests.Adversarial
+    Test.Ouroboros.Consensus.ChainGenerator.Tests.BitVector
+    Test.Ouroboros.Consensus.ChainGenerator.Tests.Counting
+    Test.Ouroboros.Consensus.ChainGenerator.Tests.Honest
     Test.Util.ChainUpdates.Tests
     Test.Util.Schedule.Tests
     Test.Util.Split.Tests
 
   build-depends:
     , base
+    , hmatrix
```

Comment:
`hmatrix` is the oddest of the new dependencies. It's used in `Test.Ouroboros.Consensus.ChainGenerator.Honest.genFirstWindowPopCount`. There are many large comments nearby that motivate the question it's used to solve.

Reply:
`hmatrix` and `genFirstWindowPopCount` were removed in 414e480. We deemed the implementation effort not worth the benefits for the time being, and we thought of simpler alternatives like generating longer chains and dropping their first windows.